cli-tests/package.json

```diff
@@ -16,7 +16,7 @@
     "@types/luxon": "^3.3.0",
     "dotenv": "^16.0.3",
     "luxon": "^3.3.0",
-    "playwright": "^1.33.0",
+    "playwright": "^1.34.0",
```

Comment:
@yurkovychv do you think `@playwright/test` and `playwright` being different versions will affect this? There's also some breaking change [here](https://github.com/microsoft/playwright/releases/tag/v1.34.0) that says "`npx playwright test` no longer works if you install both `playwright` and `@playwright/test.`"